cdap-ui/app/cdap/components/EntityListView/index.js

```diff
@@ -114,9 +114,11 @@ class EntityListView extends Component {
       showSplash: true,
       userStoreObj : '',
       notFound: false,
-      numColumns: null
+      numColumns: null,
+      total: 0
     };
 
+    // FIXME: @ajainarayanan: I'm not sure why this is outside of state :|
```

Comment:
since it was just a simple counter, nothing needed to get re-endered when this thing changed, so there was no need to put this inside a state. But perhaps that was wrong approach